docs/operator-manual/server-commands/argocd-applicationset-controller.md

```diff
@@ -60,4 +60,4 @@ argocd-applicationset-controller [flags]
       --webhook-parallelism-limit int           Number of webhook requests processed concurrently (default 50)
 ```
 
-###### Auto generated by spf13/cobra on 20-Mar-2025
+###### Auto generated by spf13/cobra on 21-Mar-2025
```

Comment:
@andrii-korotkov-verkada this line should be removed I think. After I rebased my PR, the `Check nothing has changed` step in CI is failing as this date gets updated and it complains this as a difference. I do not see this line in any other generated doc. https://github.com/argoproj/argo-cd/actions/runs/14007867560/job/39224004476?pr=21811

Reply:
The root cause is not this PR, I have created this PR to address this issue by disabling auto-generated tag in the application set CLI command. PTAL.
https://github.com/argoproj/argo-cd/pull/22446

Reply:
Caused due the change introduced in [22417](https://github.com/argoproj/argo-cd/pull/22417)

Reply:
@anandf  I think the line is coming as part of autogeneration. I will see if we can remove that line manually and run `make clidocsgen`.

Reply:
@anandf @andrii-korotkov-verkada created PR https://github.com/argoproj/argo-cd/pull/22449 to fix the issue